evaluate/metrics/assessment.go

```diff
@@ -24,26 +24,30 @@ var (
 	AssessmentKeyNoExcessResponse = RegisterAssessmentKey("no-excess-response")
 )
 
-// Assessments holds numerical assessment metrics.
+// Assessments holds a collection of numerical assessment metrics.
 type Assessments map[AssessmentKey]uint
 
-// Merge combines two assessments into a new assessment and returns it.
-func (a Assessments) Merge(o Assessments) Assessments {
-	if a == nil {
-		a = Assessments{}
-	}
-	if o == nil {
-		o = Assessments{}
-	}
+// NewAssessments create a new assessment collection.
+func NewAssessments() Assessments {
+	return map[AssessmentKey]uint{}
+}
 
-	assessments := map[AssessmentKey]uint{}
+// Add adds the given assessment collection to the current one.
+func (a Assessments) Add(x Assessments) {
+	for k, v := range x {
```

Comment:
If we don't use `allAssessmentKeys` anymore to ensure all keys are present, then we can remove it.

Reply:
For this method, we do not need it. but it is useful for doing the export of the CSV.